src/app.module.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { AppModule } from './app.module';
+
+describe('AppModule', () => {
+  it(`AppModule is successfully created`, async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    const app = moduleFixture.createNestApplication();
+    await app.init();
+    await app.close();
+  });
+});
```

Comment:
A module test can simply import the module and use it in the `App` if the module is missing a dependency it'd fail the test